common.php

```diff
@@ -161,14 +161,14 @@
 	'DIGESTS_SORT_POST_DATE'			=> 'Du plus ancien au plus récent',
 	'DIGESTS_SORT_POST_DATE_DESC'		=> 'Du plus récent au plus ancien',
 	'DIGESTS_SORT_USER_ORDER'			=> 'Selon les préférences d’affichage de mon forum',
-	'DIGESTS_SUBJECT_TITLE'				=> 'Résumé %2$s de %1$s',
+	'DIGESTS_SUBJECT_TITLE'				=> '%1$s : résumé %2$s',
 	'DIGESTS_TAG_REPLACED'				=> '<strong>Note : le contenu a été supprimé pour des raisons de sécurité. Cliquez sur le lien du message ou du sujet pour visualiser l’intégralité du contenu du message.</strong>',
 	'DIGESTS_TITLE'						=> 'Résumés',
 	'DIGESTS_TRANSLATED_BY'				=> 'Traduit par',
 	'DIGESTS_TRANSLATOR_NAME'			=> 'Philippe Bonnaure - www.macvf.fr',	// Leave null string to suppress translator name
 	'DIGESTS_TRANSLATOR_CONTACT'		=> 'https://www.macvf.fr',	// Leave null string to suppress contact info, if used use: mailto:name@emailaddress.com or a URL if you have a website.
 	'DIGESTS_TOC'						=> 'Sommaire',
```

Comment:
`Sommaire` ou `Table des matières` ?

En anglais, il a utilisé `Table of contents`

https://www.scribbr.fr/questions-frequentes/quelle-est-la-difference-entre-une-table-des-matieres-et-un-sommaire
https://www.laculturegenerale.com/difference-sommaire-table-des-matieres/

Je ne sais pas… je te laisse choisir 😄

Reply:
Les deux sont, de nos jours, à peu près interchangeables, même si je préfère "Table des matières".
Pour le coup je laisse "Sommaire", qui me semble correspondre davantage au cas d'espèce.

Reply:
Ok. On laisse "Sommaire"